koku/api/settings/tags/view.py

```diff
@@ -36,78 +30,7 @@ class SettingsTagFilter(django_filters.rest_framework.FilterSet):
     class Meta:
         model = EnabledTagKeys
         fields = ("enabled", "uuid")
-
-    def _get_order_by(
```

Comment:
This pr is mostly to move this logic to the utils so that I could use it for the aws_cost_category keys, since @samdoran and I spent time getting multiple filters functional for tags.